app/src/app/Controllers/Admin/EstadistiquesController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Controllers\Admin;
+
+use App\Core\View;
+use App\Models\Estadistiques; // Assegura't que aquest model existeix
+
+class EstadistiquesController
```

Comment:
El nom del fitxer i la classe han de ser idèntics però en anglés per seguir la nomeclatura que tenen els altres.

Et suggereixo aquest nom:
```suggestion
class StatisticsController
```